evergreen.yaml

```diff
@@ -60,11 +60,19 @@ functions:
     - command: subprocess.exec
       type: setup
       params:
+        command: make get-mongodb
         working_dir: pail
+        include_expansions_in_env: ["MONGODB_URL"]
         env:
-            MONGODB_URL: ${mongodb_url}
-            DECOMPRESS: ${decompress}
-        command: make get-mongodb
+          DECOMPRESS: ${decompress}
+    - command: subprocess.exec
+      type: setup
+      params:
+        command: make get-mongosh
+        working_dir: pail
+        include_expansions_in_env: ["MONGOSH_DOWNLOAD"]
+        env:
+          DECOMPRESS: ${decompress}
```

Comment:
It seems like `DECOMPRESS` isn't used in this target.

Reply:
Fixed